### ISSUE TITLE
invalid literal for int() 

### ISSUE THREAD

**Issue by jordibonastrem** [droefs/HLISA#35]
i get invalid literal for int() every time and im following the example :(

`    test= driver.find_element(By.XPATH,'//input[@aria-labelledby="so_mdp_hero_widget_buy_button-announce"]')
    actions.move_to_element(test_audible).click().perform()`

**Comment by basvdl97** [comment]
Hey this is most likely an error that selenium is giving you. Perhaps your issue is similar to that of this user: https://stackoverflow.com/questions/68018209/valueerror-invalid-literal-for-int-with-base-10-1-077-when-using-a-cooki

If you are convinced that it has to do with HLISE can you provide more details; such as the traceback, how you acquire `test_audible`, and perhaps what url you are working with.

**Comment by jordibonastrem** [comment]
Hello, thanks for the answer, but ive been testing all day with different websites and i always  get that error :(. 
here is a n example setup, test it if you can on your end 
`driver = webdriver.Chrome(executable_path="chromedriver.exe")
driver.get("https://www.lipsum.com/")
actions  = HLISA_ActionChains(driver)
dummy = driver.find_element(By.ID,'generate')
actions.move_to_element(dummy).perform()
actions.click()`

this throws this error:
`Exception has occurred: NoCursorCoordinatesException
invalid literal for int() with base 10: 'ove t'`

in the image below i posted the where the code fails so you can see it better
[https://ibb.co/DG6WTJY](url)

**Comment by droefs** [comment]
Thank you for reporting this issue! It is a duplicate of #31. Unfortunately, HLISA 1.5.0 has a problem with Chromium based browsers. You can either use Firefox, or you can use HLISA 1.4.1 (pip install HLISA==1.4.1) to circumvent this problem, but note that there is a bug in HLISA 1.4.1 that sometimes causes the cursor to move to an incorrect location (only after navigating to or from a different webpage.
